cmd/modulectl/create/module/cmd.go

```diff
@@ -0,0 +1,53 @@
+package module
```

Comment:
Let's move this command one hierachy higher. This cli is already intended to be used in the context of module only and therefore should look like this (linked locally as /usr/bin/module for example):
`module scaffold --etc`
`module create -- etc`
No need for the extra `module` command after `create`

Reply:
You mean `modulectl scaffold --flags-and-options` and `modulectl create --flags-and-options` I guess. Good idea, implemented.